### PR TITLE
PREVIEW-148: fixes to styles page.

### PR DESCRIPTION
**Pull request by glawson**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,30 +0,0 @@
-{
```
Comment (radfahrer) [inline bower.json]: Why nuke this file?

File: dist/css/bossy.css [hunk 2]
```diff
@@ -1,2 +1,2 @@
...id input{border-bottom-color:red}fieldset.bossy-radio,fieldset.bossy-checkbox{margin-left:-.5rem;display:flex;position:relative;padding-bottom:.25rem}fieldset.bossy-radio:after,fieldset.bossy-checkbox:after{content:"";border-bottom:solid thin #E4E3E3;display:block;position:absolute;bottom:0;left:.5rem;width:calc(100% - .5rem)}fieldset.bossy-radio label,fieldset.bossy-checkbox label{margin:0;margin-left:.5rem;white-space:nowrap;margin-top:.5rem}fieldset.bossy-radio legend,fieldset.bossy-checkbox legend{margin-top:0;margin-left:.5rem}form .group{margin-top:0}form .group fieldset{width:auto;flex:1;margin-top:0}form .group fieldset+fieldset{position:relative}form .group fieldset+fieldset:before{content:"";border-left:solid thin #E4E3E3;height:100%;display:block;position:absolute;top:0;left:0}form .group fieldset+fieldset legend,form .group fieldset+fieldset input{padding-left:.25rem}.bossy-form{padding:10px}.bossy-form form{margin:0 auto;padding:25px 50px;width:40%;min-height:115px;background:#fff;box-shadow:0 0 20px rgba(0,0,0,0.15);font:12px/15px Arial,Helvetica,sans-serif;color:#666}.bossy-form form.green{border-left:5px solid #9b2}.bossy-form form.red{border-left:5px solid #AE0002}
```
Comment (radfahrer) [inline dist/css/bossy.css]: Should this file live in the repo at all?

File: bower.json (hunk 1 above)
Comment (glawson) [inline bower.json]: Separation of the preview site from the main repo.  We will have a bower.json at the root that describes the bossyui project for bower installation.

File: dist/css/bossy.css (hunk 2 above)
Comment (glawson) [inline dist/css/bossy.css]: yeah

**Comment by glawson** [comment]
Fixes #148

**Pull request merged**